object/ldap_conn.go

```diff
@@ -559,20 +611,31 @@ func SyncUserFromLdap(organization string, ldapId string, userName string, passw
 			continue
 		}
 
-		res, _ := conn.GetLdapUsers(ldapServer, user)
+		res, err := conn.GetLdapUsers(ldapServer, user)
+		if err != nil {
+			logs.Error("get ldap users: %s", err.Error())
+
+			continue
+		}
 		if len(res) == 0 {
 			conn.Close()
 			continue
 		}
 
 		err = checkLdapUserPassword(user, password, lang)
 		if err != nil {
+			logs.Error("check ldap user password: %s", err.Error())
+
 			conn.Close()
 			return nil, err
 		}
 
 		_, _, err = SyncLdapUsers(organization, AutoAdjustLdapUser(res), ldapServer.Id)
+		if err != nil {
+			logs.Error("sync ldap users: %s", err.Error())
```

Comment:
Is added error logging written to records?  it’s not very good if some of errors is in log and some is in records

Reply:
logs is not equal records
logging is located in many places for debugging and understanding
Records only in places when it necessary
In this place record needed?